examples/go/scaleway/cdktf.json

```diff
@@ -0,0 +1,14 @@
+{
+    "language": "go",
+    "app": "go run main.go",
+    "codeMakerOutput": "generated",
+    "projectId": "8acbba0c-66a6-471b-bd44-9641bbc0f3a3",
+    "terraformProviders": [
+        "scaleway/scaleway@>= 0.13"
```

Comment:
Is there a reason you did not depend on the latest one? I couldn't even find a 0.13 version and it seems that even 1.x is already two years old 🙂

Reply:
Indeed, I copy/pasted the version I found on terraform documentation but your right, 2.2.0 version is out now :)